### DISCUSSION TITLE
2 UI Suggestions

### DISCUSSION THREAD

**Discussion by VaelophisNyx** [endless-sky/endless-sky#9322] [Ideas]
Just some suggestions but, it would be nice if the mission dialogue window wasn't left-aligned, but rather Centered. Gets a bit painful to have to look so far left on my monitor to read the text over there ^^;

and the ability to control text scaling in the mission dialogue window would be nice. Getting older, eye-sight getting worse, etc.

**Comment by kybermen** [comment]
Can I also Request a UI improvement, Can you add additional Tabs to the ship list, just to break Long List into Multiple Tabs.
Or maybe bookmarks to Jump on the list so it's easier to manage my ships. I kinda went nuts on capturing and customizing ships.

If possible do it so we can control the number of tabs. If not maybe about 10~ will do.

On how to handle it on the save file. You can just insert a special "Break line" to know where the New Tab starts on the ship list.

**Reply by mOctave** [reply]
We already have groups that you can assign your escorts to (corresponding to the number keys). It shouldn't be too hard to add that functionality to the UI plus let you see the list of escorts in each group there.
